htdocs/comm/action/index.php

```diff
@@ -919,7 +919,7 @@
 // Complete $eventarray with events coming from external module
 $parameters=array(); $object=null;
 $reshook=$hookmanager->executeHooks('getCalendarEvents',$parameters,$object,$action);
-if (! empty($hookmanager->resArray['eventarray'])) $eventarray=array_merge($eventarray, $hookmanager->resArray['eventarray']);
+if (! empty($hookmanager->resArray['eventarray'])) $eventarray=array_replace($eventarray, $hookmanager->resArray['eventarray']);
```

Comment:
I think the array_replace is not a better solution
if eventarray contains a record with key 201602 than contains 2 events, and a hook return an events for same day with 2 events, we must get on key with 4 events. 
So good fix will probably be to code a loop on $hookmanager->resArray['eventarray'] and make
$eventarray[$daykey][] = $recordinloop